frontend/src/utils/filterHelpers.js

```diff
@@ -314,6 +316,10 @@ export function isFilterValid(filters) {
 export function parseFiltersToPatch(params) {
   return params.reduce((acc, param) => {
     // filters with only defaltValue shouldn't be sent to server
+    // UPDATE: This way, when we apply lookup filters with only default values,
```

Comment:
isn't this comment (all of it) not valid anymore?

I mean, YES we shall send ALL filters back.

Abstract: the backend is advising frontend which is the default value to be used for a filter.
The frontend shall take it and use it as the initial value.

When frontend is sending back the filters, it shall send ALL of them.
I would expect at this phase to not talk about `defaultValue` but the `value` to already initialized with the user's value or with the `defaultValue` if user didn't set anything.
But that's an implementation detail.